media-plugins/vdr-osdteletext/vdr-osdteletext-0.9.7.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit user vdr-plugin-2
+
+VERSION="2161" # every bump, new version
+
+DESCRIPTION="VDR Plugin: Osd-Teletext displays the teletext on the OSD"
+HOMEPAGE="https://projects.vdr-developer.org/projects/plg-osdteletext"
+SRC_URI="https://projects.vdr-developer.org/attachments/download/${VERSION}/${P}.tgz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~x86"
```

Comment:
Where did these extra keywords come from? I can only see amd64 and x86 keyworded currently. 

You need to request re-keywording from the arch teams through bugzilla.

Rest looks good & compiles fine. I do also wonder why all these vdr packages install their own sudoers.d file.. This is just a plugin right, why does it need sudo? Can it be handled with /etc/sudoers.d/vdr from vdr package?

Reply:
> You need to request re-keywording from the arch teams through bugzilla.

Did not know that, will remove extra keywords.
 
> Rest looks good & compiles fine. I do also wonder why all these vdr packages install their own sudoers.d file.. This is just a plugin right, why does it need sudo? 

The plugin creates a cache with received teletext pages for faster display when the user selects a new page. As this cache is not needed to be written on disk, a tmpfs is created/mounted before the plugin is started. The sudo file enables the service user of the vdr process to mount  this tmpfs.
Nowadays, where /run is a tmpfs on every system I can think about re-designing this.

> Can it be handled with /etc/sudoers.d/vdr from vdr package?

It could, but this would violate the principle of minimal permissions - allow only what's really needed.
I prefer moving the teletext cache to /run, but I have to talk about it with hd_brummy.